internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -197,10 +197,18 @@ func convertAutoscaling(a manifest.AdvancedCount) (*template.AutoscalingOpts, er
 		MaxCapacity: &max,
 	}
 	if a.CPU != nil {
-		autoscalingOpts.CPU = aws.Float64(float64(*a.CPU))
+		if a.CPU.Value != nil {
+			autoscalingOpts.CPU = aws.Float64(float64(*a.CPU.Value))
+		} else {
+			autoscalingOpts.CPU = aws.Float64(float64(*a.CPU.ScaledResource.Value))
+		}
```

Comment:
Maybe it'll be safer to
```suggestion
		if a.CPU.Value != nil {
			autoscalingOpts.CPU = aws.Float64(float64(*a.CPU.Value))
		}
		if a.CPU.ScaledResource.Value != nil {
			autoscalingOpts.CPU = aws.Float64(float64(*a.CPU.ScaledResource.Value))
		}
```